api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml

```diff
@@ -18,5 +18,12 @@
 	    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
 		http://www.liquibase.org/xml/ns/dbchangelog 
 		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+	<!--
+		The empty change set is needed for integration tests. It can be removed if this file contains other change sets.
+	-->
+	<changeSet id="42ddb873-f064-4418-a6c0-6c09426c4832" author="wschlegel">
```

Comment:
Isn't it easy for one to forget removing this changeset, and in which case it ends up being released in production? Yet for the test, we do not have this risk, however small it it. 😊

Reply:
Removing the empty change set is good practice but not strictly needed. It is ok to release it to production as it does not do anything. 

The test on the other hand could slowly become out of date when the todo is not resolved. And people would need to remember to add and remove todos every time new snapshots are created.